modules/redis/redis.c

```diff
@@ -346,6 +346,13 @@ redis_dd_init(LogPipe *s)
   if (!log_dest_driver_init_method(s))
     return FALSE;
 
+  if (!self->key && !self->param1)
+    {
+      msg_error("Error initializing Redis destination: command option MUST be set!",
+                evt_tag_str("driver", self->super.super.super.id));
```

Comment:
We usually use location tags instead, they are more precise.

Reply:
Thx, fixed